src/components/Description/Description.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "./Description.css";
+
+export default class Description extends Component {
+  render() {
```

Comment:
Hai incluso `PropTypes` ma manca la dichiarazione delle props, vedi [qui](https://github.com/eclubpolimi/eclubpolimi.it/blob/084b2bacba275dad9e5d4b46cfbd8f6b4b7710a8/src/components/Timeline/Timeline.js#L6).

```suggestion
  static propTypes = {
    // title of this paragraph
    title: PropTypes.string,
    // text of this paragraph
    text: PropTypes.string,
  };
  render() {
```